CI: Add ingress/egress default deny tests for CNP

manalibhutiyani opened the pull request:
Fixes :#3343
Signed-Off-By: Manali Bhutiyani <manali@covalent.io>

Commit:
CI: Add ingress/egress default deny tests for CNP
Fixes :#3343
Signed-Off-By: Manali Bhutiyani <manali@covalent.io>

eloycoto commented on test/k8sT/Policies.go:
```diff
@@ -406,12 +411,99 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
+			Expect(err).Should(BeNil(),
+				"L3 deny-egress Policy cannot be applied in %q namespace", helpers.DefaultNamespace)
+
+			err = helpers.WaitUntilEndpointUpdates(ciliumPod, eps, 4, kubectl)
```
This `WaitUntilEndpointUpdates` can be deleted. The CiliumPolicyAction  takes care of that.

manalibhutiyani replied:
done

eloycoto commented on test/k8sT/Policies.go:
```diff
@@ -406,12 +411,99 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
+			Expect(err).Should(BeNil(),
+				"L3 deny-ingress Policy cannot be applied in %q namespace", helpers.DefaultNamespace)
+
+			err = helpers.WaitUntilEndpointUpdates(ciliumPod, eps, 4, kubectl)
```
This can be deleted as well.

manalibhutiyani replied:
done

Commit:
CI: Remove call to WaitUntilEndpointUpdates, if CiliumPolicyAction is present.
CiliumPolicyAction takes care of waiting till endpoints get updated correctly.
Remove the unnecessary calling of WaitUntilEndpointUpdates, in addition to CiliumPolicyAction.

Fixes :#3343
Signed-Off-By: Manali Bhutiyani <manali@covalent.io>
